.gitignore

```diff
@@ -94,3 +97,5 @@ jspm_packages/
 .storybook-out
 # Local Netlify folder
 .netlify
+.claude/
```

Comment:
I said I'd [vibe code](https://simonwillison.net/2025/Mar/19/vibe-coding/) this, and I'm sticking to it, no matter how much it hurts!